index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="utf-8">
+    <title>Apple Old Website Clone</title>
+    <link rel="stylesheet" href="./CSS/stylesheet.css">
+    <script src="https://kit.fontawesome.com/89c0957ca7.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <main>
+
+      <header>
```

Comment:
- [ ] Please consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document, so please take the `nav` outside of the `main` tag.